s32/drivers/s32k1/BaseNXP/header/S32K146_ADC.h

```diff
@@ -109,7 +109,8 @@ typedef struct {
   __I  uint32_t RN;                                /**< ADC Data Result Registers, offset: 0x7C */
   __I  uint32_t RO;                                /**< ADC Data Result Registers, offset: 0x80 */
   __I  uint32_t RP;                                /**< ADC Data Result Registers, offset: 0x84 */
-  __IO uint32_t CV[ADC_CV_COUNT];                  /**< Compare Value Registers, array offset: 0x88, array step: 0x4 */
+  __IO uint32_t CV1;                               /**< Compare Value Registers, offset: 0x88 */
```

Comment:
the bitfield macros for each register modified must be updated accordingly to keep a consistent header definition, even if not used by the adc12 driver

Reply:
ok

Reply:
sorry for the misunderstanding. I just commented on this line because it was the first register modified by you. What I meant is that the bitfields macros must match the register definitions. So if you change for example `ASC1x` to `SC1[]`, then the bitfields macros https://github.com/zephyrproject-rtos/hal_nxp/blob/master/s32/drivers/s32k1/BaseNXP/header/S32K146_ADC.h#L957-L973 must be changed to something like https://github.com/zephyrproject-rtos/hal_nxp/blob/master/mcux/mcux-sdk/devices/MK64F12/MK64F12.h#L457-L525. Same for any modified register.